topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md

```diff
@@ -0,0 +1,449 @@
+
+## Performing the Demultiplexing and Quantification
+
+We will now proceed to demultiplex, map, and quantify both sets of reads using the correct chemistry discovered in the previous sub-section.
+
+
+> ### {% icon hands_on %} Hands-on
+>
+> **RNA STARsolo** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"RNA-Seq FASTQ/FASTA file, cDNA reads"*: `pbmc_1k_v3_S1_L001_R1_001.fastq.gz`
+>    - {% icon param-file %} *"RNA-Seq FASTQ/FASTA file, Barcode reads"*: `pbmc_1k_v3_S1_L001_R2_001.fastq.gz`
+>    - {% icon param-repeat %} *Insert Input Pairs*
+>    - {% icon param-file %} *"RNA-Seq FASTQ/FASTA file, cDNA reads"*: `pbmc_1k_v3_S1_L002_R1_001.fastq.gz`
+>    - {% icon param-file %} *"RNA-Seq FASTQ/FASTA file, Barcode reads"*: `pbmc_1k_v3_S1_L002_R2_001.fastq.gz`
+>      (*pay attention to the* **L001** *and* **L002** *names*)
+>    - {% icon param-file %} *"RNA-Seq Cell Barcode Whitelist"*: `737K-august-2016.txt`
+>    - *"Custom or built-in reference genome"*: `Use a built-in index`
+>        - *"Reference genome with or without an annotation"*: `use genome reference without builtin gene-model`
+>            - *"Select reference genome"*: `Human (Homo Sapiens): hg19 Full`
+>            - *"Gene model (gff3,gtf) file for splice junctions"*: `Homo_sapiens.GRCh37.75.gtf`
+>    - In *"Advanced Settings"*:
+>        - *"Configure Chemistry Options"*: `Cell Ranger v3`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > We leave the *Genomic features to collect UMI counts upon* at `Gene` and *UMI deduplication (collapsing) algorithm* at `All`, as these are the options that emulate the CellRanger pipeline.
+>    >
+>    {: .comment}
+>
+{: .hands_on}
+
+
+# Producing a Quality Count Matrix
+
+The matrix files produced by **RNA STARsolo** are in the *bundled* format, meaning that the information to create a tabular matrix of Genes vs Cells are separated into different files. These files are already 10x analysis datasets, compatible with any downstream single-cell RNA analysis pipeline, however the number of cells represented here are greatly over-represented, as they have not yet been filtered for high quality cells, and therefore the matrix represents *any* cells that were unambiguously detected in the sample.
+
+If we were to construct a cell matrix using the data we have, we would have a large matrix of 60,000 Genes against 737,000 Cells, of which most values would be zero, i.e. an *extremely* sparse matrix.
+
+To get a high quality count matrix we must apply the **DropletUtils** tool, which will produce a filtered set which will produce a dataset more representative of the CellRanger pipeline.
```

Comment:
wihch will produce ... which will produce ...?

I also feel you should somewhere mention what a CellRanger pipeline is if you cite it so much.